ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -423,13 +426,11 @@ def test_ipa_healthcheck_dna_plugin_returns_warning_pagure_issue_60(self):
         # DNA configuration.
         tasks.kinit_admin(self.replicas[0])
         tasks.user_add(
-            self.replicas[0], 'ipauser1', first='Test', last='User',
+            self.replicas[0], "ipauser1", first="Test", last="User",
         )
         # Now run the ipa-healthcheck command again
         returncode, data = run_healthcheck(
-            self.replicas[0],
-            "ipahealthcheck.ipa.dna",
-            "IPADNARangeCheck",
+            self.replicas[0], "ipahealthcheck.ipa.dna", "IPADNARangeCheck",
```

Comment:
fixing formatting not relevant here.

Reply:
@rcritten the changes have been reverted.